go/unit_tests.md

```diff
@@ -43,12 +43,12 @@ Example function to be tested:
 ```go
 
 func ClusterID(obj interface{}) (string, error) {
-	customObject, ok := obj.(SpecificType)
+	customResource, ok := obj.(SpecificType)
```

Comment:
How about `cr`?

Reply:
I agree, but this is so commonly used that we all know about it and we can save some characters. Like with `ctx`.